cognite/client/utils/_client_config.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
         self.status_forcelist = self._get_status_forcelist()
         self.max_retries = int(os.getenv("COGNITE_MAX_RETRIES", 10))
         self.max_retry_backoff = int(os.getenv("COGNITE_MAX_RETRY_BACKOFF", 30))
-        self.max_connection_pool_size = int(os.getenv("COGNITE_MAX_CONNECTION_POOL_SIZE", 50))
+        self.max_connection_pool_size = int(os.getenv("COGNITE_MAX_CONNECTION_POOL_SIZE", self.max_workers))
```

Comment:
is this expected, given that someone may thread-parallelize multiple sdk calls?

Reply:
Good point. That was probably the reasoning for setting it this high to begin with.